src/lib/payment-processing/providers/radom/tokens.ts

```diff
@@ -0,0 +1,71 @@
+import type { Network, Radom } from "./types";
+
+type Tokens = {
+  [key in Network]?: Record<string, string>
+}
+
+const TestnetTokens: Tokens = {
```

Comment:
I'm sure this is done somewhere, but I've not come across it yet. Do we check the environment we're in to decide which of these arrays get used somewhere? Just want to make sure we aren't accidentally letting people pay with testnet tokens in prod (or prod in staging/local)

Reply:
That's done here: https://github.com/brave/store-brave-com/tree/crypto-payments/src/lib/payment-processing/providers/radom/adapter.ts#L53

Reply:
perfect thanks